images/encryption/encryption.go

```diff
@@ -317,7 +317,7 @@ func cryptChildren(ctx context.Context, cs content.Store, desc ocispec.Descripto
 			Layers: newLayers,
 		}
 
-		mb, err := json.MarshalIndent(newManifest, "", "   ")
```

Comment:
I think there are couple other instances of this, could you change them at the same time? Thanks!
```
cmd/ctr/commands/commands.go:181:	b, err := json.MarshalIndent(x, "", "    ")
images/encryption/encryption.go:320:		mb, err := json.MarshalIndent(newManifest, "", "   ")
images/encryption/encryption.go:400:		mb, err := json.MarshalIndent(newIndex, "", "   ")
```

Reply:
done!